packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py

```diff
@@ -234,6 +239,23 @@ def __call__(self, message: str) -> bool:
                     round(progress, 1) == 1.0
                     for progress in self._current_progress.values()
                 )
+
+        _current_timestamp = datetime.now(UTC)
+        if _current_timestamp - self._last_poll_timestamp > timedelta(seconds=5):
+            url = f"https://{self.node_id}.services.{self.get_partial_product_url()}"
+            response = httpx.get(url, timeout=10)
+            self.logger.info(
+                "Querying the service endpoint from the E2E test. Response: %s, response"
```

Comment:
MINOR: a context logger around the get call? Note that you write querying but you actually queriED